Source/WebKit/UIProcess/Cocoa/WKStorageAccessAlert.mm

```diff
@@ -27,7 +27,7 @@
 #import "WKStorageAccessAlert.h"
 
 #if PLATFORM(IOS)
-#import "UIKitSPI.h"
+#import "UIAlertControllerUtilities.h"
```

Comment:
Maybe we just move this import down below, since the rest of the code is compiled out otherwise?